torch/_inductor/codegen/cpp.py

```diff
@@ -2079,7 +2086,7 @@ def to_dtype(x, dtype):
                             ):
                                 # 1. doing uint8 to float.
                                 # 2. the previous node of uint8 to float is load.
-                                opt_ctx.is_load_uint8_as_float = True
+                                pass
                             elif dtype in [torch.bfloat16, torch.float]:
                                 pass
```

Comment:
combine these two if checks?

Reply:
Sure, combine uint8 with bfloat16 in this case.